apps/shopify/src/skuResolvers.js

```diff
@@ -133,7 +133,7 @@ export const fetchProductPreviews = async (skus, config) => {
   }
 
   const response = (await Promise.all(requests)).flat();
-  const products = response.map((res) => convertProductToBase64(res));
+  const products = response.map((res) => res && convertProductToBase64(res));
```

Comment:
Couldn't trigger an error with this line of code. Mine reverting this back to how it was?
```suggestion
  const products = response.map((res) => convertProductToBase64(res));
```

Reply:
For Products, In-app Configuration, we have two options - Product and Product Variant, With Variant option It is working fine but when we select Product option then it throws an error

Reply:
Sounds good! I see what you mean now! Mind making the PR to this repo: https://github.com/contentful/marketplace-partner-apps

We've moved the shopify app's code over here! I am having trouble applying your commit over there and want you to receive credit

Reply:
I have opened a [PR](https://github.com/contentful/marketplace-partner-apps/pull/101) in the [new repo](https://github.com/contentful/marketplace-partner-apps). you can review it